gitlab/application.yaml

```diff
@@ -0,0 +1,159 @@
+---
+# note to the installing admin:
+# this app needs a couple prereq steps
+# 1. pull request and atlantis apply a new oidc client named gitlab so the secrets are available in vault
+# 2. adjust the helm.values.global.hosts.name name to attach it to your domain
+
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: gitlab
+  namespace: argocd
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+  annotations:
+    argocd.argoproj.io/sync-wave: '10'
```

Comment:
any problem if other part of application has the same wave? or just apply for manifest?
![CleanShot 2024-01-17 at 14 54 19](https://github.com/kubefirst/gitops-catalog/assets/3932407/bb1e5a21-1b68-43fd-99f3-dafa13f8dad5)

Reply:
yeah no issue with these all being on wave 10. that basically just means they all have an equal opportunity to sync without prerequesite steps.